packages/lesmis/src/views/MapView.js

```diff
@@ -17,27 +33,137 @@ const Container = styled.div`
   min-height: 600px;
 `;
 
+const Map = styled(MapContainer)`
+  flex: 1;
+  height: 100%;
+`;
+
 const Main = styled.div`
   position: relative;
   flex: 1;
   height: 100%;
 `;
 
+const EmptyPanel = styled(MapOverlaysPanelContainer)`
+  padding: 2rem;
+`;
+
+const Legend = styled(LegendComponent)`
+  position: absolute;
+  bottom: 5px;
+  z-index: 999;
+  left: 50%;
+  transform: translateX(-50%);
+`;
+
+const TilePicker = styled(TilePickerComponent)`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  z-index: 9999;
+  pointer-events: none;
+`;
+
+const useProvinceData = () => {
+  const query = useEntitiesData();
+  let provinces = [];
+
+  if (query.data) {
+    provinces = query.data.filter(entity => entity.parentCode === 'LA');
+  }
+
+  return { ...query, data: provinces };
+};
+
 export const MapView = () => {
   const { entityCode } = useUrlParams();
+  const [activeTileSetKey, setActiveTileSetKey] = useState(TILE_SETS[0].key);
+  const [hiddenMeasures, setHiddenMeasures] = useState([]);
   const [selectedOverlay, setSelectedOverlay] = useState(null);
 
+  useEffect(() => {
+    setHiddenMeasures([]);
+  }, [setHiddenMeasures, selectedOverlay]);
+
+  const { data: entityData } = useEntityData({
+    entityCode,
+  });
+
+  const { data: overlayReportData } = useMapOverlayReportData({
+    entityCode,
+    mapOverlayCode: selectedOverlay,
+    hiddenMeasures,
+  });
+
   const { data: overlaysData, isLoading } = useMapOverlaysData({ entityCode });
 
+  const { data: provicesData } = useProvinceData();
```

Comment:
```suggestion
  const { data: provincesData } = useProvinceData();
```

Reply:
And maybe also 
```suggestion
  const { data: provincesData } = useProvincesData();
```
so they have consistent pluralisation

Reply:
Also, considering we're not using the loading state at all, it could even be 
```suggestion
  const provinces = useProvinces();
```
which would also make the hook itself simpler.

On that note - _should_ we be using the loading state of any of these other queries?